item-impl/src/main/java/com/example/auction/item/impl/PItemCommand.java

```diff
@@ -43,6 +41,92 @@ public int hashCode() {
         }
     }
 
+
+    final class UpdateItem implements PItemCommand, PersistentEntity.ReplyType<PUpdateItemResult> {
+        private final UUID id;
+        private final UUID creator;
+        final String title;
+        private final String description;
+        private final String currencyId;
+        private final int increment;
+        private final int reservePrice;
+        private final Duration auctionDuration;
+
+        @JsonCreator
+        public UpdateItem(UUID id, UUID creator, String title, String description, String currencyId, int increment, int reservePrice, Duration auctionDuration) {
```

Comment:
`CreateItem` has the advantage that is only handled when there's no `PItemState`. Updating, OTH, needs a previous `PItemState` (with it's internal `PItem`) and, on top of that, can happen on `create` and on `auction` behaviours.